lib/tasks/local_authorities.rake

```diff
@@ -0,0 +1,6 @@
+namespace :local_authorities do
+  desc "Import local authority names, codes and tiers from MapIt"
+  task import_codes: :environment do
```

Comment:
This task is importing more than just codes - it's the full list of all LAs and determines which ones exist for us. Although we import other data about LAs from other sources, those tasks don't create new authorities, they only update particular attributes. Can we find a name which makes it clearer that this is the main one, maybe `import_list`?

The description could also make it clearer that this is the one which does the creating by starting "Import the list of..."  rather than just "Import...", and then the descriptions for the slug and homepage URL tasks could be "Add [slugs/homepage URLs] to local authorities from [source]" which would also clarify the relationships between the tasks.